src/components/author-primitives/packer/checklist/style.module.css

```diff
@@ -6,13 +6,15 @@
 .root {
 	& ul {
 		list-style: none;
-		margin-left: 2rem;
+		margin-left: 8px;
```

Comment:
@braydenlove let me know what spacing you'd like here. I tried to take reference from the regular list item margin.

Reply:
I just commented, thanks again!